csharp/HOLMS.Platform/HOLMS.Platform/Types/Extensions/IAM/Department.cs

```diff
@@ -1,45 +1,29 @@
 ﻿namespace HOLMS.Types.IAM {
     public partial class Department {
-        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capStaffMgmt, bool capGroupMgmt,
-            bool capHousekeepingMgmt, bool capAccountingReports, bool capManagementReports,
-            bool capWorkstationParameters, bool capSaveDisplaySettings, bool capLodgeicReset,
-            bool capEventsWaitlistSetup, bool capSystemSettings, bool capYieldManagement,
-            bool capHoldMgmt, bool capGuestBlacklistAddRemove, bool capOverbookRooms,
-            bool capOverrideRates, bool capCallAccountingAdvancedSetup, bool capGiftShopsalesSetup,
-            bool capCreditCardSetup, bool capCreditCardWorksheet, bool capCreditCardCaptureSettle,
-            bool capCreditCardReporting, bool capExpediaQCMgmt, bool capWebSetup) {
+        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capGetSupplySnapshot, bool capGetRoomTypes,
```

Comment:
Agree that it sucks. what about just keeping a repeated SecurityAction enum on the dto? it's not like monolith / lodgeic can't figure that out? and lodgeic would just map that to the existing set of options on the display?